curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c5f81f0cf325b4a854c.md

```diff
@@ -7,34 +7,34 @@ dashedName: step-4
 
 # --description--
 
-Within the `body` element, create a `div` with the `class` set to `header`.
+Within the `body` element, create a `header` element. Give the `header` element a class with `header` as the value.
```

Comment:
I would avoid using the word nest as people often do not understand what that means. Part of the change is about code blocks (as explained).

Reply:
How about (not sure why the body is now tags instead of an element?)
```
Add a `header` element within the body tags. Give it `header` as the class.
```

Or 
```
Add a `header` element within the body element and assign the class `header` to it.
```

Reply:
I changed my suggestion to something close to your second.